ibmcloud/README.md

```diff
@@ -215,32 +158,6 @@ clean state:
 - If using a self-managed cluster you can delete the whole cluster following the
 [Delete the cluster documentation](./cluster#delete-the-cluster) and then start again.
 - If you instead just want to leave the cluster, but uninstall the Confidential Containers and peer pods
-feature, the following commands can be run:
-  - Delete the cloud-api-adaptor pods with:
-    ```
-    kubectl delete -k install/overlays/ibmcloud
-    ```
-  - Delete the confidential containers controller manager with:
-    ```
-    kubectl delete -f install/yamls/deploy.yaml
-    ``` 
-    - Wait for the all the confidential containers pods to to finish terminating and be removed with:
-    ```
-    kubectl get pods -n confidential-containers-system --watch
-    ```
-  - Delete the webhook with:
-    ```
-    kubectl delete -f ./webhook/hack/webhook-deploy.yaml
-    ``` 
-    - Check that the webhook pod have been removed with:
-    ```
-    kubectl get pods -n peer-pods-webhook-system
-    ```
-  - Delete cert-manager with:
-    ```
-    kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/v1.9.1/cert-manager.yaml
-    ```
-    - Check that the cert-manager pods have been removed with:
-    ```
-    kubectl get pods -n cert-manager
-    ```
\ No newline at end of file
+feature, you can use the `caa-provisioner-cli` to remove the resources.
+
+`caa-provisioner-cli -action=uninstall`
```

Comment:
Some clarity of where to run this from would probably be good here as well

Reply:
I got a failure during clean-up in the script, but it did work:
```
./caa-provisioner-cli -action=uninstall
INFO[0000] IamServiceURL is: https://iam.cloud.ibm.com/identity/token.
INFO[0000] VpcServiceURL is: https://ca-tor.iaas.cloud.ibm.com/v1.
INFO[0000] Uninstalling CoCo operator and cloud-api-adaptor resources
INFO[0000] Uninstall CoCo and cloud-api-adaptor
FATA[0001] no matches for kind "CcRuntime" in version "confidentialcontainers.org/v1beta1"
```

```
$ kubectl get pods -A
NAMESPACE      NAME                                                        READY   STATUS    RESTARTS   AGE
kube-flannel   kube-flannel-ds-flwjn                                       1/1     Running   0          4d19h
kube-flannel   kube-flannel-ds-g28dd                                       1/1     Running   0          4d19h
kube-system    coredns-5d78c9869d-gmpz9                                    1/1     Running   0          4d19h
kube-system    coredns-5d78c9869d-p426w                                    1/1     Running   0          4d19h
kube-system    etcd-sh-sm-z-cluster-2023-05-10-node-0                      1/1     Running   0          4d19h
kube-system    ext-res-updater-6q59x                                       1/1     Running   0          3d21h
kube-system    kube-apiserver-sh-sm-z-cluster-2023-05-10-node-0            1/1     Running   0          4d19h
kube-system    kube-controller-manager-sh-sm-z-cluster-2023-05-10-node-0   1/1     Running   0          4d19h
kube-system    kube-proxy-5l7r7                                            1/1     Running   0          4d19h
kube-system    kube-proxy-pntsc                                            1/1     Running   0          4d19h
kube-system    kube-scheduler-sh-sm-z-cluster-2023-05-10-node-0            1/1     Running   0          4d19h
```

It's probably not a defect on this PR, but something to be aware of

Reply:
Again, I don't think this appeared in my testing but if can be recreated should be looked at